src/vanilla/utils/unwrap.ts

```diff
@@ -95,10 +95,10 @@ export function unwrap<Value, Args extends unknown[], Result, PendingValue>(
       return atom(
         (get) => {
           const state = get(promiseAndValueAtom)
-          if ('v' in state) {
-            return state.v
+          if ('f' in state) {
+            return state.f
```

Comment:
It looks a little odd that f(allback) seems to have priority over v(alue).

Reply:
Yeah, but with 'f' and 'v', it means 'v' is obsolete.

```js
if ('v' in state && !('f' in state)) {
  return state.v
}
return state.f
```
that requires more bytes.